components/nav.js

```diff
@@ -1,39 +1,36 @@
 import Link from "next/link";
 import { useTheme } from "next-themes";
 import { CgDarkMode } from "react-icons/cg";
+import { FiGithub } from "react-icons/fi";
 
-const links = [
-    { label: "Page 1", href: "/" },
-    { label: "Page 2", href: "/" },
-    { label: "Page 3", href: "/" }
-];
+import SearchBar from "./searchbar";
 
 export default function Nav() {
     const { theme, setTheme } = useTheme();
     return (
         <nav className="dark:text-white">
-            <ul className="flex flex-wrap sm:justify-between items-start sm:items-center p-8 mt-6 sm:mt-0">
-                <li>Perfect Next.js Theme</li>
+            <ul className="flex flex-wrap sm:justify-between items-start sm:items-center">
                 <ul className="mx-auto sm:mx-0 flex flex-row space-x-5">
-                    {links.map(({ href, label }) => (
-                        <li className="self-center" key={`${href}${label}`}>
-                            <Link href={href}>
-                                <a className="font-inter px-4 py-2 rounded hover:bg-black dark:hover:bg-white hover:bg-opacity-10 dark:hover:bg-opacity-10">
-                                    {label}
-                                </a>
-                            </Link>
-                        </li>
-                    ))}
                     <li>
+                        <SearchBar />
+                    </li>
+                    <li className="self-center">
+                        <Link href="https://github.com/thinkfiveable/open">
+                            <a className="p-3 block rounded-full hover:bg-black dark:hover:bg-white hover:bg-opacity-10 dark:hover:bg-opacity-10 transform duration-200">
```

Comment:
what does this transform do here? there's nothing happening that needs this class